src/ct/ct_pref_dlg_theme.cc

```diff
@@ -93,7 +93,7 @@ Gtk::Widget* CtPrefDlg::build_tab_theme()
     auto label_style_scheme_co = Gtk::manage(new Gtk::Label{_("Code")});
     auto combobox_style_scheme_co = Gtk::manage(new Gtk::ComboBoxText{});
     for (auto& scheme : _pCtMainWin->get_style_scheme_manager()->get_scheme_ids()) {
-        if (not Glib::str_has_prefix(scheme, "user-")) {
```

Comment:
This is wrong, we have user-1 and user-2 to filter out

Reply:
You're right there! Didn't track that there are two style schemes

Reply:
Sorry but this is not my commit. I added the following

260        pButtonsResetToDefault[i]->signal_clicked().connect([this, i, pConfig,
261                                                             pColorButtonTextFg,
262                                                             pColorButtonTextBg,
263                                                             pColorButtonSelectionFg,
264                                                             pColorButtonSelectionBg,
265                                                             pColorButtonCursor,
266                                                             pColorButtonCurrentLineBg,
267                                                             pColorButtonLineNumbersFg,
268                                                             pColorButtonLineNumbersBg,
269                                                             f_onUserStyleChanged](){
270            bool anyChange{false};
271            if (CtDialogs::question_dialog(reset_warning, *this)) {